File: vue/src/store/entities/customer.ts
import Entity from './entity'

export default class Customer extends Entity<number>{
    studentName?: string
    mother?: string
    father?: string
    fatherPhone?: string
    motherPhone?: string
    otherGuardian?: string
    otherGuadianPhone?: string
    note?: string
    state:string
    lastModificationTime?: string
    lastModifierUserId?: number
    creationTime?: string
    creatorUserId?: number
}

File: aspnet-core/src/EducationAdmin.Core/Sales/Customer.cs
using Abp.Domain.Entities;
using Abp.Domain.Entities.Auditing;
using System;
using System.Collections.Generic;
using System.Text;

namespace EducationAdmin.Sales
{
    public class Customer : FullAuditedEntity<long>, IMustHaveTenant
    {

        public string StudentName { get; set; }


        public string Mother { get; set; }

        public string Father { get; set; }

        public string FatherPhone { get; set; }

        public string MotherPhone { get; set; }

        public string OtherGuardian { get; set; }


        public string OtherGuadianPhone { get; set; }

        public string Note { get; set; }

        public CustomerState State { get; set; }
        public int TenantId { get; set; }
    }

    public enum CustomerState
    {
        Created=0,

        FollowUp=1,

        Droped=2,

        Success=3
    }
}
